storybookjs/storybook
# Addon-Storyshots: Adding web-component support

Issue: currently Storyshots can not be used with web-components because Storyshots does not ship with a web-components loader

## What I did

Copied over the HTML loader and changed the appropriate bits to make it load for web-components.

## How to test

All the current Storyshot stories and tests seem to be testing Storyshot functionality against React. There are no existing tests for HTML so I did not add any explicit tests for web-components. Existing tests all pass, but don't really test this new functionality.

- Is this testable with Jest or Chromatic screenshots? n/a
- Does this need a new example in the kitchen sink apps? n/a
- Does this need an update to the documentation? n/a

If your answer is yes to any of these, please make sure to include it in your PR.
